src/components/Login.jsx

```diff
@@ -25,26 +23,28 @@ const Login = props => {
       .signOut()
       .then(() => {
         props.changeAuth(false);
-        setDisplayLoginButton(true);
+        props.changeLoginButton(true)
+        props.changeSignupButton(true);
       })
       .catch(error => {
-        setLoginMessage(error);
+        props.changeAuthMessage(error);
       });
   };
 
   let loginFunction;
 
   switch (true) {
-    case displayLoginButton && !props.authenticated:
+    case props.displayLoginButton && !props.authenticated && props.displaySignupButton:
```

Comment:
Is displaySignupButton ever true while displayLoginButton is false? Otherwise it seems redundant to have them both

Reply:
It's needed to not display the login button when Signup is false.